students/kmsnyde/lesson06/test_mailroom4.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Apr 28 13:19:54 2018
+
+@author: Karl M. Snyder
+"""
+
+"""
+Define tests that will work againsts mailroom4.py.
+"""
+
+import mailroom4 as mail
+import os.path
+
+def test_1():
+    donors = mail.donor_dict
+    assert "Karl Stick" in donors
```

Comment:
This test is not necessary since it tests that defaultdict works as expected and there's no need to test Python